syntax = "proto3";

package NSound.NClient;

import "protos/common/property-list.proto";

message TContextMessage {

    message TConnect {
        string name = 1;
        NCommon.TPropertyList property_list = 2;
    }

    oneof request {
        TConnect connect = 1;
    }

}